app/dashboard/tip_views.py

```diff
@@ -283,7 +283,8 @@ def send_tip_3(request):
             primary_email = to_emails['events'][0]
         else:
             print("TODO: no email found.  in the future, we should handle this case better because it's GOING to end up as a support request")
-
+    if type(primary_email) is list and len(primary_email):
```

Comment:
We can omit `len()` and `type() is` in favor of `if primary_email and isinstance(primary_email, list):` - checks for val followed by the type assertion.

Reply:
dbec78ef4edab0ce4e996426e44d847cc6c2d813